src/ontology/uberon-edit.obo

```diff
@@ -224679,6 +224679,14 @@ is_a: UBERON:0000464 ! anatomical space
 relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
 property_value: dcterms-date 2022-07-05T10:46:32Z xsd:dateTime
 
+[Term]
+id: UBERON:8480014
+name: interlobular stroma of mammary gland
+def: "The interlobular stroma of the mammary gland lobule separates the lobules and intralobular stroma from any substantial direct contact with the adipose tissue. It is more compact than the intralobular stroma." [PMID:18710550]
```

Comment:
I think it makes sense to add the axiom:
'adjacent to' some 'intralobular stroma of mammary gland'
This will make the two structures spatially disjoint which I think is helpf